### ISSUE TITLE
Can't open project in Visual Studio

### ISSUE THREAD

**Issue by 0x53A** [fable-compiler/fable-powerpack#60]
![image](https://user-images.githubusercontent.com/4236651/48743068-d091d080-ec61-11e8-929e-dac6d3f94574.png)

This helped:

![image](https://user-images.githubusercontent.com/4236651/48743085-e30c0a00-ec61-11e8-833f-0731e7f49bbf.png)

But I have no idea if that has any negative side effects.

**Comment by alfonsogarciacaro** [comment]
Yes, I think I had a similar project. Some months ago, Nuget wouldn't package the subfolders correctly if the slash was omitted (that's the reason of the comment), but maybe they've fixed it. I'd have to check.

**Comment by 0x53A** [comment]
Hm, can you maybe try ``extra\**`` ?

**Comment by alfonsogarciacaro** [comment]
Sorry, forgot to check with VS 😅 The package is being split into standalone modules #63 So we don't need to worry about this anymore :)
